pootle/apps/pootle_app/management/commands/update_tmserver.py

```diff
@@ -73,7 +57,121 @@ def _parse_translations(self, **options):
                 'store__translation_project__language__code'
             ).order_by()
 
-        total = units_qs.count()
+        return units_qs.iterator(), units_qs.count()
+
+    def get_unit_data(self, unit):
+        """Return dict with data to import for a single unit."""
+        fullname = (unit['submitted_by__full_name'] or
+                    unit['submitted_by__username'])
+
+        email_md5 = None
+        if unit['submitted_by__email']:
+            email_md5 = md5(unit['submitted_by__email']).hexdigest()
+
+        return {
+            "_index": self.INDEX_NAME,
+            "_type": unit['store__translation_project__language__code'],
+            "_id": unit['id'],
```

Comment:
I know this is a copy&paste, however I'd advise against mixing quoting in the same dict.